kodilla-stream/src/test/java/com/kodilla/stream/world/WorldTestSuite.java

```diff
@@ -0,0 +1,71 @@
+package com.kodilla.stream.world;
+import org.junit.Assert;
+import org.junit.Test;
+import java.math.BigDecimal;
+
+public class WorldTestSuite {
+
+    @Test
+    public void testGetPeopleQuantityOfWorld() {
+        //given
+        Country japonia = new Country("Japonia", new BigDecimal("127000000"), "Asia");
```

Comment:
when using numbers (not strings) you can write them in a following way: 127_000_000 - then it is easier to read.